packages/server/src/service/signup.ts

```diff
@@ -0,0 +1,66 @@
+import { StateInfo, ContactMethod } from '../common'
```

Comment:
I think this is from a separate commit.  Shall we wait to rebase this once this has been merged?

Reply:
It's a bit different from the changes from #133, this is based on the work at #172 (which this PR is built upon). Without this we don't truly get all the failures from `./mg.ts`